### DISCUSSION TITLE
Help to calibrate a MOES curtain module (Solved)

### DISCUSSION THREAD

**Discussion by PacmanForever** [Koenkk/zigbee2mqtt#12520] [Devices]
Hi everyone,

I have this module:

https://www.zigbee2mqtt.io/devices/MS-108ZR.html

But I don't know how to calibrate it in HA or Z2M Web UI.

I have others similiar modules and all has a calibration switch.

Could someone help me please?

Thanks

**Comment by holgerschwenk** [comment]
Hi, same here. I didnˋt find the switch. How was this solved?
Thanks

**Reply by PacmanForever** [reply]
Using Zigbee2Mqtt web UI

![imagen](https://github.com/Koenkk/zigbee2mqtt/assets/37216513/b8fa8e54-a5fa-46d6-8b2a-26bc645f277c)
